torch/_decomp/decompositions.py

```diff
@@ -3434,6 +3434,26 @@ def arange_start(
     )
 
 
+@register_decomposition(aten.__and__)
+def dunder_and(self, value):
+    return torch.bitwise_and(self, value)
+
+
+@register_decomposition(aten.__or__)
+def dunder_or(self, value):
+    return torch.bitwise_or(self, value)
+
+
+@register_decomposition(aten.__lshift__)
+def dunder_lshift(self, value):
+    return torch.bitwise_left_shift(self, value)
+
+
+@register_decomposition(aten.__rshift__)
+def dunder_rshift(self, value):
```

Comment:
Given that they have `out` variants, decorate them with the `out_wrapper` decorator.

Reply:
Fixed.